Lib/multiprocessing/managers.py

```diff
@@ -417,7 +425,11 @@ def decref(self, c, ident):
             return
 
         with self.mutex:
-            assert self.id_to_refcount[ident] >= 1
+            if self.id_to_refcount[ident] <= 0:
+                raise ProcessError(
```

Comment:
I would keep it an assert (or an AssertionError), since this really seems to be testing an internal consistency thing.